scripts/west_commands/runners/x86_configuration.txt

```diff
@@ -0,0 +1,3 @@
+power_off = curl http://admin:1234@192.168.1.20/script?run003=run
+tftp_root_dir = ~/vftp/kernel
+power_on = curl http://admin:1234@192.168.1.20/script?run001=run
```

Comment:
This is very custom and random without any explanation what that is and what lies behind it. Need to think about how to make this generic and remove the need for that, maybe add a command line option that invokes a script that lives out of the tree to reboot the target, without having all of this in the tree.

Reply:
@nashif  yes, I can add an argument behind west flash command using this runner's do_add_parser function, whick will set the script location, in this way,  the west flash --option works well.
So if I want to run sanitycheck, I also have to add this option to sanitycheck via parser.add_argument in sanitycheck script, and pass the added option to the final west flash command in devicehandler function. I have to modify two place in the sanitycheck script, do you think this is an reasonable way?